R/utils.R

```diff
@@ -75,6 +75,7 @@ source_files = function(format = NULL, config = load_config(), all = FALSE) {
     if (!is.na(index)) files = c(files[index], files[-index])
   }
   check_special_chars(files)
+  unique(normalize_path(files))
```

Comment:
We need to keep the file paths relative and should not resolve them to absolute paths. But you are right that we should make sure the file paths are unique.